crates/redstone/src/core/config.rs

```diff
@@ -58,12 +68,16 @@ impl Config {
         signers: Vec<SignerAddress>,
         feed_ids: Vec<FeedId>,
         block_timestamp: TimestampMillis,
+        max_timestamp_delay_ms: TimestampMillis,
```

Comment:
minor: I think we want 2 creating functions now:
* try_new same signature as before and uses const for ahead and delay timestamps
* try_new_with_timestamps = current try_new within this pr

Reply:
What about `Option<T>` in the same function?